src/lftp_ssl.cc

```diff
@@ -863,6 +863,7 @@ lftp_ssl_openssl::~lftp_ssl_openssl()
 {
    SSL_free(ssl);
    ssl=0;
+   global_deinit();
```

Comment:
I don't think we need to deinit the whole openssl every time a connection is closed. A better solution is needed.

Reply:
It just needs to be done before exit. Can you suggest a better location?

Reply:
I think OpenSSL clean up can be skipped altogether, it is not strictly required.

Reply:
It may not be strictly required, but the documentation does not suggest it is optional.

From
https://www.openssl.org/docs/man1.1.1/man3/OPENSSL_init_crypto.html

OPENSSL_INIT_NO_ATEXIT

    By default OpenSSL will attempt to clean itself up when the process exits via an "atexit" handler. Using this option suppresses that behaviour. This means that the application will have to clean up OpenSSL explicitly using OPENSSL_cleanup().